modules/installation-guide/partials/con_persistent-volume-configuration.adoc

```diff
@@ -5,95 +5,127 @@
 [id="storage-strategies-for-{prod-id-short}-workspaces_{context}"]
 = Storage strategies for {prod-id-short} workspaces
 
-Workspace Pods use Persistent Volume Claims (PVCs), which are bound to the physical Persistent Volumes (PVs) with link:https://kubernetes.io/docs/concepts/storage/persistent-volumes/#access-modes[ReadWriteOnce access mode]. It is possible to configure how the {prod-short} server uses PVCs for workspaces. The individual methods for this configuration are called PVC strategies:
+Storage strategy::
+A configurable method defining how {prod-short} workspaces uses persistent volume claims (PVCs) and persistent volumes (PVs). 
```

Comment:
```suggestion
A configurable method defining how {prod-short} workspaces uses persistent volume claims (PVCs) which are bound to the physical persistent volumes (PVs).
```

Reply:
I think it does not make much sense to the user to introduce details on the relationship between PV and PVC.